Source/WebCore/css/parser/CSSPropertyParserConsumer+Grid.cpp

```diff
@@ -148,10 +152,10 @@ bool parseGridTemplateAreasRow(StringView gridRowNames, NamedGridAreaMap& gridAr
         currentColumn = lookAheadColumn - 1;
     }
 
+    gridAreaMap.rowCount++;
```

Comment:
Our usual style is this:
```suggestion
    ++gridAreaMap.rowCount;
```